lib/taiko.js

```diff
@@ -2431,6 +2431,81 @@ function fileField(attrValuePairs, _options = {}, ...args) {
   return elemWrapper;
 }
 
+/**
+ * This {@tableCell selector} lets you identify a table cell
+ * on a web page with row and column and row values as options and locating table using proximity selectors,
+ * or table labels.
+ *
+ * @example
+ * tableCell({row:1, col:1}, "Table Caption")
+ * tableCell({},{id:'myColumn'}).text()
```

Comment:
Is there someway to make this api easier?
For example 
```
tableCell({id:'myColumn'}).text()
```

Instead of passing and empty `{}`?

Reply:
Added!